web/src/main/java/se/inera/intyg/webcert/web/service/access/DraftAccessServiceImpl.java

```diff
@@ -207,7 +206,7 @@ public AccessResult allowToReadyForSign(AccessEvaluationParameters accessEvaluat
             .careUnit(accessEvaluationParameters.getUnit())
             .patient(accessEvaluationParameters.getPatient())
             .checkPatientDeceased(true)
-            .invalidCertificateTypeForDeceased(DbModuleEntryPoint.MODULE_ID)
+            .excludeCertificateTypesForDeceased(DbModuleEntryPoint.MODULE_ID)// You should be able to sign a draft for a deceased person
```

Comment:
.invalidCertificateTypeForDeceased() affects every call to this method regardless of AccessEvaluationParameters. This may affect other calls to this method if it's removed. .excludeCertificateTypesForDeceased only affects deceased persons.

Reply:
Same goes for ReadyForSign as for forwarding.

By adding excludeCertificateTypesForDeceased you will allow the user to do this.